api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirTaskDaoImpl.java

```diff
@@ -55,7 +47,7 @@ public FhirTask saveTask(FhirTask task) throws DAOException {
 	}
 	
 	@Override
-	public FhirTask getTaskByUuid(String uuid) {
+	public FhirTask get(String uuid) {
 		return (FhirTask) sessionFactory.getCurrentSession().createCriteria(FhirTask.class).add(eq("uuid", uuid))
 		        .uniqueResult();
 	}
```

Comment:
You should be able to get rid of this function altogether

Reply:
Finished